source/common/modules/markdown-editor/index.ts

```diff
@@ -94,6 +93,8 @@ export default class MarkdownEditor extends EventEmitter {
     this._cmOptions = getCodeMirrorDefaultOptions(this)
     this._countChars = false
 
+    this._md2html = getConverter((window as any).getCitation)
```

Comment:
Would `window.getCitation` also work?

Reply:
Hah, yes! That's the last API function I haven't yet typed! Thanks for finding it! 😍